src/Form/Form.js

```diff
@@ -118,23 +118,20 @@ class Form {
 
         this.logFormInfo()
 
-        const numKnownInputs = this.inputs.all.size - this.inputs.unknown.size
-        if (numKnownInputs === 0) {
-            // This form has too few known inputs and likely doesn't make sense
-            // to track for autofill (e.g. a single-input for search or item quantity).
-            // Self-destruct to stop listening and avoid memory leaks.
-            if (shouldLog()) {
-                console.log(`Form discarded: zero inputs are fillable`)
-            }
-            this.destroy()
-            return
-        }
-
         if (shouldAutoprompt) {
             this.promptLoginIfNeeded()
         }
     }
 
+    get hasOnlyUnknownFields () {
+        const numKnownInputs = this.inputs.all.size - this.inputs.unknown.size
+
+        if (numKnownInputs === 0) {
+            return true
+        }
+        return false
+    }
+
```

Comment:
This is the same logic as before, but now it's moved so that the scanner can ask the question when it chooses to.